package.rmd

```diff
@@ -291,6 +291,6 @@ When you use `library(pkg)` to load a package, R looks through each path in `.li
 library(blah)
 ```
 
-The main difference between `library()` and `require()` is what happens when a package isn't found. While `library()` throws an error, `require()` prints a message and returns FALSE. In practice this distinction isn't important because when building a package you should __NEVER__ use either inside a package. See [package dependencies](#dependencies) for what you should do instead.
+As stated above, the main difference between `library()` and `require()` is what happens when a package isn't found. While `library()` throws an error, `require()` prints a message and returns FALSE. In practice this distinction isn't important because when building a package you should __NEVER__ use either inside a package. See [package dependencies](#dependencies) for what you should do instead.
```

Comment:
The difference between `library()` and `require()` is stated twice in a short span of text in, I felt the phrasing should reflect this, but it's obviously a question of style preferences.